e2e_samples/parking_sensors/databricks/notebooks/01_explore.py

```diff
@@ -3,9 +3,12 @@
 import datetime
 
 # For testing
-base_path = 'dbfs:/mnt/datalake/data/lnd/2019_10_06_05_54_25'
-parkingbay_filepath = os.path.join(base_path, "ParkingLocationData.json")
-sensors_filepath = os.path.join(base_path, "ParkingSensorData.json")
+# Replace values before testing
+stgaccountname=""
+existing_data=""
+base_path = f"abfss://datalake@{stgaccountname}.dfs.core.windows.net/data/lnd/{existing_data}"
+parkingbay_filepath = os.path.join(base_path, "MelbParkingBayData.json")
```

Comment:
Why are we pointing to MelbParkingBayData.json still? I think these files have been removed from the project. See the reference to 02_standardize.py using the new parking data.